gossip/keys.go

```diff
@@ -45,6 +45,9 @@ const (
 	// KeyConfigPermission is the permission configuration map.
 	KeyConfigPermission = "permissions"
 
+	// KeyConfigUser is the user configuration map.
+	KeyConfigUser = "user"
```

Comment:
Yup. done. I've noticed a bunch of refactoring while digging in this PR. #1548 and #1549 filed against myself.